packages/artillery-engine-playwright/index.js

```diff
@@ -74,7 +82,7 @@ class PlaywrightEngine {
                   name: metric.name,
                   value: metric.value,
                   metric: metric,
-                  url: window.location.href
+                  url: window.location.href // eslint-disable-line
```

Comment:
curious - what does eslint complain about here? (and elsewhere)

Reply:
It complains about this:
```
   85:24  error    'window' is not defined               no-undef
```

I'll disable the specific rule actually on all of these, to make it more clear 👍

Reply:
It still complains about `webVitals` btw:
`   80:13  error    'webVitals' is not defined            no-undef`

But for that one there's no way for eslint to disable it due to being inline, IIRC. We'll have to move the webVitals script to its own file at some point in another PR to address this